process.c

```diff
@@ -5588,54 +5604,54 @@ proc_getrlimit(VALUE obj, VALUE resource)
- *  [RTTIME] CPU time for real-time process (us) (GNU/Linux)
- *  [SBSIZE] all socket buffers (bytes) (NetBSD, FreeBSD)
- *  [SIGPENDING] number of queued signals allowed (signals) (GNU/Linux)
- *  [STACK] stack size (bytes) (SUSv3)
- *
- *  _cur_limit_ and _max_limit_ may be
- *  <code>:INFINITY</code>, <code>"INFINITY"</code> or
- *  Process::RLIM_INFINITY,
- *  which means that the resource is not limited.
- *  They may be Process::RLIM_SAVED_MAX,
- *  Process::RLIM_SAVED_CUR and
- *  corresponding symbols and strings too.
- *  See system setrlimit(2) manual for details.
- *
- *  The following example raises the soft limit of core size to
- *  the hard limit to try to make core dump possible.
+ *    Process.setrlimit(resource, cur_limit, max_limit = cur_limit) -> nil
+ *
+ *  Sets limits for the current process for the given +resource+
+ *  to +cur_limit+ (soft limit) and +max_limit+ (hard limit);
+ *  returns +nil+.
+ *
+ *  Argument +resource+ specifies the resource whose limits are to be set;
+ *  the argument may be given as a symbol, as a string, or as a constant
+ *  beginning with <tt>Process::RLIMIT_</tt>
+ *  (e.g., +:CORE+, <tt>'CORE'</tt>, or <tt>Process::RLIMIT_CORE</tt>.
+ *
+ *  The resources available and supported are system-dependent,
+ *  and may include (here expressed as symbols):
+ *
+ *  - +:AS+: Total available memory (bytes) (SUSv3, NetBSD, FreeBSD, OpenBSD except 4.4BSD-Lite).
+ *  - +:CORE+: Core size (bytes) (SUSv3).
+ *  - +:CPU+: CPU time (seconds) (SUSv3).
+ *  - +:DATA+: Data segment (bytes) (SUSv3).
+ *  - +:FSIZE+: File size (bytes) (SUSv3).
+ *  - +:MEMLOCK+: Total size for mlock(2) (bytes) (4.4BSD, GNU/Linux).
+ *  - +:MSGQUEUE+: Allocation for POSIX message queues (bytes) (GNU/Linux).
+ *  - +:NICE+: Ceiling on process's nice(2) value (number) (GNU/Linux).
+ *  - +:NOFILE+: File descriptors (number) (SUSv3).
+ *  - +:NPROC+: Number of processes for the user (number) (4.4BSD, GNU/Linux).
+ *  - +:NPTS+: Number of pseudo terminals (number) (FreeBSD).
+ *  - +:RSS+: Resident memory size (bytes) (4.2BSD, GNU/Linux).
+ *  - +:RTPRIO+: Ceiling on the process's real-time priority (number) (GNU/Linux).
+ *  - +:RTTIME+: CPU time for real-time process (us) (GNU/Linux).
+ *  - +:SBSIZE+: All socket buffers (bytes) (NetBSD, FreeBSD).
+ *  - +:SIGPENDING+: Number of queued signals allowed (signals) (GNU/Linux).
+ *  - +:STACK+: Stack size (bytes) (SUSv3).
+ *
+ *  Arguments +cur_limit+ and +max_limit+ may be:
+ *
+ *  - Integers (+max_limit+ should not be smaller than +cur_limit+).
+ *  - Symbol +:SAVED_MAX+, string <tt>'SAVED_MAX'</tt>,
+ *    or constant <tt>Process::RLIM_SAVED_MAX: saved maximum limit.
+ *  - Symbol +:SAVED_CUR+, string <tt>'SAVED_CUR'</tt>,
+ *    or constant <tt>Process::RLIM_SAVED_CUR: saved current limit.
+ *  - Symbol +:INFINITY+, string <tt>'INFINITY'</tt>,
+ *    or constant <tt>Process::RLIM_INFINITY: no limit on resource.
```

Comment:
I think you're missing the closing `</tt>` tag here.

```suggestion
 *  - Integers (+max_limit+ should not be smaller than +cur_limit+).
 *  - Symbol +:SAVED_MAX+, string <tt>'SAVED_MAX'</tt>,
 *    or constant <tt>Process::RLIM_SAVED_MAX</tt>: saved maximum limit.
 *  - Symbol +:SAVED_CUR+, string <tt>'SAVED_CUR'</tt>,
 *    or constant <tt>Process::RLIM_SAVED_CUR</tt>: saved current limit.
 *  - Symbol +:INFINITY+, string <tt>'INFINITY'</tt>,
 *    or constant <tt>Process::RLIM_INFINITY</tt>: no limit on resource.
```